### DISCUSSION TITLE
Tauri Board Election 2023

### DISCUSSION THREAD

**Discussion by Beanow** [tauri-apps/governance-and-guidance#31] [General]
Hello @tauri-apps/working-group as mentioned in [a blog earlier](https://tauri.app/blog/2023/06/15/tauri-board-elections-and-governance-updates) we're organizing an election for the Tauri Board this year where anyone part of the Working Group can participate in the vote!

Voting has started and you can vote **until Wednesday July 12th, 2023**.

Information about the election is collected here:
**[`governance-and-guidance/board-election-2023/README.md`](https://github.com/tauri-apps/governance-and-guidance/blob/main/board-election-2023/README.md)**

Besides the written introductions here, we're inviting candidates for an Ask Me Anything on our Discord.
Keep an eye out for more information about this soon.

Feel free to generally ask questions or discuss the 2023 Tauri Board elections in this thread. (In addition to our Discord channels.)

**Comment by beanow-at-crabnebula** [comment]
Happy Monday all! For those subscribed here, just a reminder you can vote until **this Wednesday** July 12th, 2023.
The AMA is taking place on [`#wg-forum/Tauri Board Candidate AMA`](https://discord.com/channels/616186924390023171/1127526517287690300).

Your vote is important. So I hope you'll take a moment of time to decide and cast your vote.
More details and how to vote available on:
https://github.com/tauri-apps/governance-and-guidance/blob/main/board-election-2023/README.md

**Comment by Beanow** [comment]
@tauri-apps/working-group The results are in!
https://github.com/tauri-apps/governance-and-guidance/blob/main/board-election-2023/README.md
